helix-core/src/test/java/org/apache/helix/integration/messaging/TestBatchMessage.java

```diff
@@ -264,10 +274,16 @@ public void testSubMsgExecutionFail() throws Exception {
     Map<String, Map<String, String>> errStates = new HashMap<String, Map<String, String>>();
     errStates.put("TestDB0", new HashMap<String, String>());
     errStates.get("TestDB0").put(errPartition, masterOfPartition0);
-    boolean result =
-        ClusterStateVerifier.verifyByPolling(new ClusterStateVerifier.BestPossAndExtViewZkVerifier(
-            ZK_ADDR, clusterName, errStates));
-    Assert.assertTrue(result);
+
+    ClusterStateVerifier.BestPossAndExtViewZkVerifier verifier = new ClusterStateVerifier.BestPossAndExtViewZkVerifier(
```

Comment:
nit, why the other BestPossAndExtViewZkVerifier does not need the prefix "ClusterStateVerifier."?

Reply:
Removed ClusterStateVerifier from this one too.